lib/rubocop/cop/tokens_util.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+module RuboCop
+  # Common methods and behaviors for dealing with tokens.
+  module TokensUtil
+    module_function
+
+    # rubocop:disable Metrics/AbcSize
+    def tokens(node)
+      @tokens ||= {}
```

Comment:
I realize you just copy-pasted, but `{}.compare_by_identity` would be nicer to use.